web/wp-config.php

```diff
@@ -18,7 +18,7 @@
  * Fetch .env
  */
 if ( ! isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && file_exists( $rootPath . '/.env' ) ) {
-	$dotenv = new Dotenv\Dotenv( $rootPath );
+	$dotenv = Dotenv\Dotenv::create($rootPath);
```

Comment:
Changes to Dotenv too in a major version bump. I tested the `.env` file locally and it works as expected.